File: src/main/java/net/eduard/economy/command/EconomyReloadCommand.java
package net.eduard.economy.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.economy.EduEconomy;

public class EconomyReloadCommand extends CommandManager {

	public EconomyReloadCommand() {
		super("reload", "recarregar");
		setDescription("Recarregar o sistema de economia");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		EduEconomy.getInstance().reload();
		sender.sendMessage("§aRecarregando sistema");

		return true;
	}

}

File: src/main/java/net/eduard/economy/command/EconomyReloadCommand.kt
package net.eduard.economy.command

import net.eduard.api.lib.manager.CommandManager
import net.eduard.economy.EduEconomyPlugin
import org.bukkit.command.CommandSender

class EconomyReloadCommand : CommandManager("reload", "recarregar") {

    override fun command(sender: CommandSender, args: Array<String>) {
        EduEconomyPlugin.instance.reload()
        sender.sendMessage(EduEconomyPlugin.instance.message("system-reload"))
    }

    init {
        description = "Recarregar o sistema de economia"
    }
}